app/Http/Controllers/VotesController.php

```diff
@@ -204,4 +205,36 @@ public function removeVote(Request $request, $id)
         $this->destroyVote($id);
         return AdministrationController::votes();
     }
-}
+
+    public function checkFakeEmails()
+    {
+        $api_key = env('API_KEY');
+        $api_url = 'https://emailverification.whoisxmlapi.com/api/v2';
+
+        $votes = Vote::all();
+
+        foreach ($votes as $vote) {
+            $response = Http::get($api_url, [
+                'apiKey' => $api_key,
+                'emailAddress' => $vote['voter_email'],
+                'format' => 'json'
+            ]);
+
+            if ($response->ok()) {
+                $data = $response->json();
```

Comment:
What about `$vote->isFake = $data['smtpCheck' ]`

Reply:
Could be